collect_app/src/main/res/layout/form_chooser_list_item_text.xml

```diff
@@ -34,7 +34,7 @@
 
     <TextView
         android:id="@+id/form_update_alert"
-        android:layout_width="wrap_content"
+        android:layout_width="match_parent"
```

Comment:
Why do we need this change?

Reply:
Some proportion of the text was cut because of this.